prac_10/app.py

```diff
@@ -0,0 +1,30 @@
+from flask import Flask
+
+app = Flask(__name__)
+
+
+@app.route('/')
+def hello_world():
+    return 'Hello World!'
+
+
+@app.route('/greet')
+@app.route('/greet/<name>')
+def greet(name=""):
+    return f"Hello {name}"
+
+
+@app.route('/c/<fahrenheit>')
+def convert_to_celsius(fahrenheit):
+    celsius = 5 / 9 * (float(fahrenheit) - 32)
+    return str(celsius)
+
+
+@app.route('/f/<celsius>')
+def convert_to_fahrenheit(celsius):
```

Comment:
I think you need to write "celsius= 22(as an example)" .

Reply:
Got it, I will fix that, thanks! ^^